content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,235 @@
+## Pre-requisites
```

Comment:
@JST5000 Can you guide me on how I can compile this md file to view it in the browser. Was having trouble when I followed the links you provided beforehand.

Reply:
Hmmm... I also can't seem to figure out where the tutorial is generated. @mDuo13 do you know if there's a special command to generate tutorial files like `build-a-desktop-wallet-in-python.md`? (After doing `dactyl_build` I'm unable to find a file with the same name, or even the same text inside of it)

Reply:
(As an aside, if you're using VSCode, I just found out about the Live Server extension and it seems pretty handy for viewing HTML files - https://stackoverflow.com/a/48721885 )

Reply:
> Hmmm... I also can't seem to figure out where the tutorial is generated. @mDuo13 do you know if there's a special command to generate tutorial files like `build-a-desktop-wallet-in-python.md`? (After doing `dactyl_build` I'm unable to find a file with the same name, or even the same text inside of it)

What machines do you guys use? After installing some libraries (xml, etc.) it works flawlessly on a Mac M1.

Reply:
@TusharPardhe ok, figured it out. 

You need to do two things:

1. Add the tutorial to `dactyl-config.yml` **(Screenshot of how attached)**
2. Add the header from the [comment below](https://github.com/XRPLF/xrpl-dev-portal/pull/1752#discussion_r1119446181) to your tutorial.md file (copied from the Python tutorial w/ the name changed from "Python" to "JavaScript")

Then when you run `dactyl_build` I see this error, which seems to mean it's actually compiling it now :)
```
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dactyl/page.py", line 218, in preprocess
    md = self.pp_template.render(**context, **self.filter_exports())
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "content/tutorials/build-apps/build-a-desktop-wallet-in-js.md", line 98, in top-level template code
  File "/Users/jacksonmills/Documents/bounty_submissions/tushar-dev-portal/tool/filter_include_code.py", line 9, in include_code
    with open("content/"+filename, encoding="utf-8") as f:
Preprocessor error in page <Page 'Pre-requisites' (build-a-desktop-wallet-in-js.html)>: [Errno 2] No such file or directory: 'content/_code-samples/build-a-wallet/js/src/helpers/render-xrp-logo.js'.
```

<img width="807" alt="image" src="https://user-images.githubusercontent.com/19694186/221956587-14aeb3b7-bf5e-40fc-b6bd-66802053081b.png">

Reply:
@AlexanderBuzz I think what happened for me was I didn't delete the `out` folder, and it didn't fully update for whatever reason. After clearing out that file, I was able to generate the Python version, and after following the above steps it seemed to include the js version as well (although the build failed)

Reply:
Thanks @AlexanderBuzz I was able to build the webpage.

@JST5000 I forgot to push the changes that's why the build failed, please try again it will work now :)